models/qrCodeAuth.js

```diff
@@ -0,0 +1,34 @@
+const firestore = require("../utils/firestore");
+const qrAuthModel = firestore.collection("Qr_auth");
+
+const updateStatus = async (userId, authStatus = "NOT_INIT") => {
+  try {
+    const authData = await qrAuthModel.doc(userId).get();
+
+    if (!authData) {
+      return {
+        userExists: false,
+      };
+    }
+
+    await qrAuthModel.doc(userId).set({
+      ...authData.data(),
+      authStatus,
```

Comment:
it is not `authStatus` we are returning `authorization_status`, according to the contract.
jut make it
```
...set({
 ...authData.data(),
    authorization_status : authStatus,
})
```